developer_guides/unit_tests.rst

```diff
@@ -84,6 +84,30 @@ Preparing cmocka package
 
       ./configure --with-cmocka-prefix=/home/<you>/cminstall ...
 
+Wraping objects for unit tests
+******************************
+
+If at any point should You need to mock any symbol it can be done by simply defining it in unit test and only including .h files. However there can exist corner cases in which function that should be mocked can't be done in this way. The cases are - Static functions in headers(those most probably are inline short functions and don't have to be mocked) or static functions that are in the same file as tested functionality and are exceedingly large so they can't be tested as one functionality. Whatever the reason mocking of those symbols can be done by using --wrap linker functionality. To wrap the symbol follow these steps:

```

Comment:
Line 90 - "If you need to mock a symbol, define it in a unit test and include the .h file. There are 2 cases where this isn't possible:

* Static functions in headers...
* Static functions that are ...

Whatever the reason ...